source/indexes/compound-index.txt

```diff
@@ -0,0 +1,95 @@
+.. _kotlin-sync-compound-index:
+
+================
+Compound Indexes
+================
+
+.. contents:: On this page
+   :local:
+   :backlinks: none
+   :depth: 2
+   :class: singlecol
+
+.. facet::
+   :name: genre
+   :values: reference
+ 
+.. meta::
+   :keywords: index, query, optimization, efficiency
+
+Overview
+--------
+
+:manual:`Compound indexes </core/index-compound/>` hold references to multiple
+fields within a collection's documents, improving query and sort performance.
+
+When creating a compound index, you must specify the following:
+
+- The fields on which to create the index
+- The sort order for each field (ascending or descending)
+
+Sample Data
+~~~~~~~~~~~
+
+The examples in this guide use the ``movies`` collection in the ``sample_mflix``
+database from the :atlas:`Atlas sample datasets </sample-data>`. To learn how to create a
+free MongoDB Atlas cluster and load the sample datasets, see the
+:atlas:`Get Started with Atlas </getting-started>` guide.
+
+The following {+language+} data class models the documents in this collection:
+
+.. literalinclude:: /includes/indexes/indexes.kt
+   :start-after: start-movie-class
+   :end-before: end-movie-class
+   :language: kotlin
+   :copyable:
+
+Create a Compound Index
+-----------------------
+
+The following example creates a compound index on the ``type`` and ``genre`` fields, with
+both fields indexed in ascending order:
+
+.. literalinclude:: /includes/indexes/indexes.kt
+   :start-after: start-index-compound
+   :end-before: end-index-compound
+   :language: kotlin
+   :copyable:
+   :dedent:
+
+The following is an example of a query that uses the index created in
+the preceding code example:
```

Comment:
[s] Maybe this introduction can be reworded to improve clarity? It can be hard to parse "following example" vs "preceding example" all in one sentence.